.github/workflows/ci.yaml

```
@@ -0,0 +1,65 @@
+name: CI
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        name: PHPUnit Tests
+        strategy:
+            fail-fast: false
+            matrix:
+                php-versions: [ '7.2', '7.3', '7.4', '8.0', '8.1', 'nightly' ]
```

```suggestion
                php: [ '7.2', '7.3', '7.4', '8.0' ]
```

Let's reduce the verbosity of the matrix variables a bit.

And let's limit the CI to the stable PHP versions for now. We can add PHP 8.1 and 8.2 in a follow-up if we like.

---

Done